### ISSUE TITLE
The contents of `:host()` and `:host-context()` are not considered

### ISSUE THREAD

**Issue by mattrbeck** [keeganstreet/specificity#54]
E.g. `:host(.x)` is calculated as 0-1-0 but should be 0-2-0.

**Comment by prantlf** [comment]
Yes. `@bramus/specificity` computes the specificity of this selector well. And the version 0.4.1 of this package computed it well too.
